src/core/stripAndCollapse.js

```diff
@@ -0,0 +1,12 @@
+define( function() {
+	"use strict";
+
+	// Strip and collapse whitespace according to HTML spec
+	// https://html.spec.whatwg.org/multipage/infrastructure.html#strip-and-collapse-whitespace
+	var rhtmlSpace = /[\x20\t\r\n\f]+/g,
```

Comment:
> Also it might be a tab.

Not sure to what you referring to.

For the reference canonical way is to use ` `, don't have strong opinion about it though